src/main/groovy/edu/oregonstate/mist/locations/mapper/LocationMapper.groovy

```diff
@@ -132,12 +133,16 @@ class LocationMapper {
         buildResourceObject(facilLocation.calculateId(), attributes)
     }
 
-    private void setLinks(ResourceObject resourceObject) {
+    private void setLinks(ResourceObject resourceObject, String locId) {
         String resource = isService(resourceObject.attributes) ? Constants.SERVICES :
                 Constants.LOCATIONS
 
         String selfUrl = "$apiEndpointUrl$resource/${resourceObject.id}"
         def links = ['self': selfUrl]
+        if(resourceObject.attributes.type == "dining" && locId != null) {
```

Comment:
Could you use this constant instead? https://github.com/osu-mist/locations-api/blob/76156c43e90ce81994c17af19a44d01bc1187b3f/src/main/groovy/edu/oregonstate/mist/locations/Constants.groovy#L6